libs/components/core/src/lib/modules/dock/dock.module.ts

```diff
@@ -2,6 +2,7 @@ import { NgModule } from '@angular/core';
 
 /**
  * @deprecated The `SkyDockModule` is no longer needed and can be removed from your application.
+ * @internal
```

Comment:
This module is empty and deprecated, and we don't want it to appear on the docs.